contracts/near/eth2-client/src/tests/integration_sepolia.rs

```diff
@@ -0,0 +1,152 @@
+#[cfg(test)]
+mod sepolia_integration_tests {
+    use super::*;
+    use crate::tests::utils::get_sepolia_test_data;
+    use crate::tests::utils::InitOptions;
+    use eth2_utility::types::InitInput;
+    use eth_types::eth2::{ExtendedBeaconBlockHeader, SyncCommittee};
+    use eth_types::{Address, Bloom, H256, H64, U256};
+    use borsh::{BorshDeserialize, BorshSerialize};
+    use near_sdk::{Gas, NearToken};
+    use near_workspaces::{Account, Contract};
+    use near_workspaces::operations::Function;
+    use serde::{Deserialize, Serialize};
+
+    const WASM_FILEPATH: &str = "../target/near/eth2_client/eth2_client.wasm";
+    const WASM_V_0_1_0_FILEPATH: &str = "src/data/eth2_client_v0.1.0_testnet.wasm";
+
+    #[derive(Debug, Clone, BorshSerialize, BorshDeserialize, Serialize, Deserialize)]
+    struct BlockHeaderV1 {         
```

Comment:
No need for a separate file for Sepolia tests, the tests should be implemented once for testnet and mainnet, the difference should be just the data